forge-gui/res/cardsfolder/d/doomsday_confluence.txt

```diff
@@ -1,7 +1,7 @@
 Name:Doomsday Confluence
 ManaCost:X X B
 Types:Sorcery
-A:SP$ Charm | CharmNum$ X | Choices$ DBSac,DBToken,DBDiscard | CanRepeatModes$ True
+A:SP$ Charm | CharmNum$ X | Announce$ X | Choices$ DBSac,DBToken,DBDiscard | CanRepeatModes$ True
```

Comment:
any reason why this is necessary?

also what about the others with `CharmNum$ X` ?